OWNERS_ALIASES

```diff
@@ -5,7 +5,6 @@ aliases:
     - EmilienM
     - Fedosin
```

Comment:
While we're at it, we should also remove Fedosin and adduarte from this file.

Reply:
also added @stephenfin and sorted alphabetically.